src/System.Globalization/ref/System.Globalization.Forwards.cs

```diff
@@ -17,3 +17,4 @@
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Globalization.RegionInfo))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Globalization.StringInfo))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Globalization.TextElementEnumerator))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Globalization.ISOWeek))]
```

Comment:
Why is this needed?

Reply:
Related, which reference assembly is this being exposed from?  Presumably from System.Runtime, as that's where Calendar now is:
https://github.com/dotnet/corefx/blob/824af7f2ba689479767cddc33741cf030650d47e/src/System.Runtime/ref/System.Runtime.cs#L4096
so System.Runtime.cs should be updated to include the surface area for this new type.

Reply:
yes we should expose this from System.Runtime 

Something like the following:

[https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L4096](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L4096)

---
In reply to: [195475409](https://github.com/dotnet/corefx/pull/30386#discussion_r195475409) [](ancestors = 195475409)

Reply:
we should remove this from System.Globalization. having it in System.Runtime should be enough. and if we need to expose it later from any other contract, we can do it in System.Globalization.Calendars one. but I don't think we need to do it. 

cc @weshaggard

---
In reply to: [195474093](https://github.com/dotnet/corefx/pull/30386#discussion_r195474093) [](ancestors = 195474093)